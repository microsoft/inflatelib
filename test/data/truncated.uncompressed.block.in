
# This generates a file with a single uncompressed block whose reported size is one greater than the number of bytes available
# NOTE: This test is valid for both Deflate and Deflate64
>1
1       # bfinal = true since this is the only block
00      # No compression

# NOTE: This will byte align, which is desired
>16
007A    # LEN = 122
FF85    # NLEN = ~LEN

# And finally, the data
>8
"This string is only 121 characters even though the file reports a length of 122 characters, meaning we will never get EOF"
