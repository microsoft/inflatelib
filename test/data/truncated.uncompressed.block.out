
"This string is only 121 characters even though the file reports a length of 122 characters, meaning we will never get EOF"
